dimension/dimension_test.go

```diff
@@ -20,11 +21,16 @@ import (
 	"github.com/ONSdigital/dp-dataset-api/url"
 	dprequest "github.com/ONSdigital/dp-net/v2/request"
 	"github.com/gorilla/mux"
-	. "github.com/smartystreets/goconvey/convey"
+	"github.com/smartystreets/goconvey/convey"
 )
 
 var (
-	urlBuilder            = url.NewBuilder("localhost:20000")
+	codeListAPIURL, _     = neturl.Parse("http://localhost:22400")
+	datasetAPIURL, _      = neturl.Parse("http://localhost:22000")
+	downloadServiceURL, _ = neturl.Parse("http://localhost:23600")
+	importAPIURL, _       = neturl.Parse("http://localhost:21800")
+	websiteURL, _         = neturl.Parse("http://localhost:20000")
+	urlBuilder            = url.NewBuilder(websiteURL, downloadServiceURL, datasetAPIURL, codeListAPIURL, importAPIURL)
```

Comment:
As mentioned in a previous comment - If this set of URL's is now being parsed using a function, a check on the error response should be made and not ignored. Previously the websiteURL was just being passed in as a string.